### ISSUE TITLE
Budgie Desktop: Missing styling for the places indicator applet

### ISSUE THREAD

**Issue by cybre** [adapta-project/adapta-gtk-theme#462]
**Reporter info**

 ```
 * Distribution - Solus
 * Gtk+ 3.0 version - 3.24
 * Desktop environment - Budgie
 * Display server protocol - X11
 ```

**Adapta version (if you knew)**
Latest git

**Related Application and/or Shell theming (or name of widget)**
Budgie Places Indicator

**Actual issue**
The CSS hasn't been updated for the new BudgiePopover type.

**Comment by tista500** [comment]
So what is the "ACTUAL' issue?

**Comment by cybre** [comment]
Sorry, I was looking at the wrong tree.
![screenshot from 2017-07-22 00 06 47](https://user-images.githubusercontent.com/3101505/28484284-c19216d6-6e71-11e7-9588-727fddde0e86.png)
The header (which is now a button) lacks any padding.

**Comment by tista500** [comment]
`.places-section-header`, right?

**Comment by cybre** [comment]
Yes. .places-section-header > button.
I deleted my comment because I thought I was tripping for a moment then I realised I was on the wrong commit again.

**Comment by tista500** [comment]
Commit: https://github.com/adapta-project/adapta-gtk-theme/commit/e36d982ee839bfde5c75fd051454a1dcb72313dd

That is our current styling for PlacesIndicator's BudgiePopoverWindow.
GtkBox contains the 'expanded' GtkButton as a child to open revealer, but for us, it should be styled similar with 'activatable' GtkListBoxRow. So I did not change that button spacing dramatically but I polished GtkLabel and GtkImage node a bit instead... ;)

Regards.
